pkg/apis/pipeline/v1alpha1/task_validation_test.go

```diff
@@ -105,7 +105,7 @@ func TestTaskSpec_Validate(t *testing.T) {
 				Name:       "mystep",
 				Image:      "${inputs.resources.foo.url}",
 				Args:       []string{"--flag=${inputs.params.baz} && ${input.params.foo-is-baz}"},
-				WorkingDir: "/foo/bar/${outputs.resources.source}",
+				WorkingDir: "/foo/bar/${outputs.resources.source.name}",
```

Comment:
This was actually an example of https://github.com/tektoncd/pipeline/issues/627. I had to modify this test in order for it to pass with the fix from this PR.